plugins/suggested-blocks/README.md

```diff
@@ -17,29 +17,39 @@ npm install @blockly/suggested-blocks --save
 ## Usage
 ```js
 import * as Blockly from 'blockly';
-import {toolboxCategories} from '@blockly/dev-tools';
 import * as SuggestedBlocks from '@blockly/suggested-blocks';
 
-function getNewCategories() {
-  const mostUsedCategory = '<category name="Frequently Used" custom="MOST_USED"></category>';
-  const recentlyUsedCategory = '<category name="Recently Used" custom="RECENTLY_USED"></category>';
-
-  // This type of insertion works in a pinch, but you would more so want to add
-  // the new categories wherever the rest of the categories are defined in the
-  // project (e.g. HTML, or a dedicated file for all the categories)
-  const indexToInsert = toolboxCategories.indexOf('</xml>');
-  // Insert the new categories into the existing toolbox XML string
-  return toolboxCategories.slice(0, indexToInsert) + mostUsedCategory + recentlyUsedCategory + toolboxCategories.slice(indexToInsert);
-}
+const toolbox = {
+  "kind": "categoryToolbox",
+  "contents": [
+    {
+      "kind": "category",
+      "name": "My Category",
+      "contents": [ /* your category contents */ ]
+    },
+    {
+      'kind': 'category',
+      'name': 'Frequently Used',
+      'custom': 'MOST_USED',
+      'categorystyle': 'frequently_used_category',
+    },
+    {
+      'kind': 'category',
+      'name': 'Recently Used',
+      'custom': 'RECENTLY_USED',
+      'categorystyle': 'recently_used_category',
+    },
+  ]
+};
+```
```

Comment:
Whoops, looks like you added the backticks in the wrong place. The backticks should be after the entire code sample, the same as before. I believe they should be after "SuggestedBlocks.init(workspace);"

Reply:
@divziv there's more [information about backticks/code fences](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) on the GitHub docs, if you're curious.